model/derived/gasoutputs.go

```diff
@@ -12,6 +12,7 @@ import (
 
 type GasOutputs struct {
 	tableName          struct{} `pg:"derived_gas_outputs"` //nolint: structcheck,unused
+	Height             int64    `pg:",pk,use_zero,notnull"`
```

Comment:
We will need to backfill the value for this column.

Reply:
The value corresponds to the height of the message I think (not that of the receipts).

Reply:
Yes, it's the height of the message. Backfilling is a problem. With the current version it will take some time. With the new in-order processing it takes <1s per tipset, but that's still ~211000 seconds = ~2.4 days unless we shard the processing.

An alternative is to figure out a query that can update this table by joining with messages. That will still be slow to run, several hours at least.

Reply:
Custom in-order processing that just does messages and just updates this should be faster (?). That said, 2.4 days should be ok.